src/ol_infrastructure/components/aws/database.py

```diff
@@ -52,7 +85,100 @@ def is_valid_version(cls: 'OLDBConfig', engine_version: Text, values: Dict) -> T
         return engine_version
 
 
-class AmazonPostgresDB(pulumi.ComponentResource):
+class OLPostgresDBConfig(OLDBConfig):
+    engine: Text = 'postgres'
+    engine_version: Text = '12.3'
+    port: PositiveInt = PositiveInt(5432)
+    parameter_overrides: List[Dict[Text, Union[Text, bool, int, float]]] = [
+        {'name': 'client_encoding', 'value': 'UTF-8'},
+        {'name': 'server_encoding', 'value': 'UTF-8'},
+        {'name': 'log_timezone', 'value': 'UTC'},
+        {'name': 'timezone', 'value': 'UTC'},
+        {'name': 'rds.force_ssl', 'value': 1}
+    ]
+
+
+class OLMariaDBConfig(OLDBConfig):
+    engine: Text = 'mariadb'
+    engine_version: Text = '10.4.8'
+    port: PositiveInt = PositiveInt(3306)
+    parameter_overrides: List[Dict[Text, Union[Text, bool, int, float]]] = [
```

Comment:
Is ssl on by default in here unlike Postgres?

Reply:
So, whether TLS is used looks to be a property of how the client connects and there doesn't seem to be an option to force TLS.